vivid-server/src/models/user-update.dto.ts

```diff
@@ -0,0 +1,3 @@
+export class UserUpdateDto {
+  name: string;
```

Comment:
You can add a decorator from `class-validator` package to constrain the string like:
```ts
@IsNotEmpty()
name: string;
```